### ISSUE TITLE
[bug] Fishing is not giving any rewards.

### ISSUE THREAD

**Issue by FrozenBeard** [Rocologo/MobHunting#313]
I have set the rewards to drop 100% of the time when catching fish, but it never, ever triggers.

```
# ########################################################################
# Rewards for fishing
# ########################################################################
# Here is where you set the base prize in $ for catching a fish
fishing:

  # Set this to true if you want to disable all fishing rewards / features.
  disable-fishing-rewards: false
  rawfish: '0'
  rawfish-cmd: mobhunt head give {player} rawfish Raw_Fish 1 silent
  rawfish-cmd-desc: You got the head of a Raw Fish
  rawfish-cmd-run-chance: 1.0
  rawfish-head-prize: '0'
  rawsalmon: '0'
  rawsalmon-cmd: mobhunt head give {player} rawsalmon Raw_Salmon 1 silent
  rawsalmon-cmd-desc: You got the head of a Raw Salmon
  rawsalmon-cmd-run-chance: 1.0
  rawSalmon-head-prize: '0'
  clownfish: '0'
  clownfish-cmd: mobhunt head give {player} clownfish Clowfish 1 silent
  clownfish-cmd-desc: You got the head of a Clownfish
  clownfish-cmd-run-chance: 1.0
  clownfish-head-prize: '0'
  pufferfish: '0'
  pufferfish-cmd: mobhunt head give {player} pufferfish Pufferfish 1 silent
  pufferfish-cmd-desc: You got the head of a Pufferfish
  pufferfish-cmd-run-chance: 1.0
  pufferfish-head-prize: '0'
```

**Comment by Rocologo** [comment]
Not confirmed. It works on my Test server. Try use /mh debug

And let me see the output in the console?

**Comment by FrozenBeard** [comment]
its detecting the fishing, the fishing mob being caught, but it is not giving me a head!

```
[16:28:06] [Server thread/INFO]: [MobHunting] Debugmode enabled. Check server console.
[16:28:07] [Server thread/INFO]: Maximus_Terragon issued server command: /top
[16:28:10] [Server thread/INFO]: [MobHunting][Debug] FishingEvent: State=FISHING
[16:28:13] [Server thread/INFO]: Shiver me timbers! Unknown Command Landlubber, try "/info" and "/help"
[16:28:16] [Server thread/INFO]: xMadMonkeyx issued server command: /c left side of the wall
[16:28:16] [Server thread/INFO]: [BE] xMadMonkeyx » left side of the wall
[16:28:22] [Server thread/INFO]: xMadMonkeyx issued server command: /c but not in the tower :/
[16:28:22] [Server thread/INFO]: [BE] xMadMonkeyx » but not in the tower :/
[16:28:26] [Server thread/INFO]: Browe_ issued server command: /c hm
[16:28:26] [Server thread/INFO]: [BE] Browe_ » hm
[16:28:29] [Server thread/INFO]: JavaInvader issued server command: /home castle
[16:28:34] [Server thread/INFO]: JavaInvader issued server command: /kit boatswain
[16:28:46] [Async Chat Thread - #76/INFO]: [BE] KapitanKirn » Hes in that building
[16:28:49] [Server thread/INFO]: [MobHunting][Debug] FishingEvent: State=BITE
[16:28:50] [Server thread/INFO]: [MobHunting][Debug] FishingEvent: State=CAUGHT_FISH, GodsDead caught a RAW_FISH(0)
[16:28:50] [Server thread/INFO]: [MobHunting][Debug] Basic Prize=£0 for catching a RawFish
[16:28:59] [Server thread/INFO]: Bi0logisch issued server command: /r the right one is better
[16:28:59] [Server thread/INFO]: Browe_ issued server command: /c ok we need to go closer
[16:28:59] [Server thread/INFO]: [BE] Browe_ » ok we need to go closer
[16:29:01] [Async Chat Thread - #76/INFO]: [BE] KapitanKirn » But the cannons would hit him i think
[16:29:01] [Server thread/INFO]: [MobHunting] Debugmode disabled.
```

**Comment by Rocologo** [comment]
Bug confirmed and fixed in build 484.

https://fractal.lindegaard.one:8181/job/MobHunting/

Please comfirm :-)

**Comment by Rocologo** [comment]
Should be fixed in 5.1.5
